src/main.tsx

```diff
@@ -1,6 +1,9 @@
 import React, { Suspense, lazy } from "react";
 import ReactDOM from "react-dom/client";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import "primereact/resources/primereact.min.css";
+import "primereact/resources/themes/saga-blue/theme.css";
```

Comment:
use this theme instead.
import 'primereact/resources/themes/lara-light-indigo/theme.css';